packages/webapp/src/containers/Finances/NewExpense/ExpenseCategories/index.jsx

```diff
@@ -80,90 +61,37 @@ class ExpenseCategories extends Component {
-                width: '100%',
-              }}
-            >
-              {expenseTypes
-                ?.sort((firstExpenseType, secondExpenseType) => {
-                  if (firstExpenseType.expense_translation_key === 'OTHER') return 1;
-                  if (secondExpenseType.expense_translation_key === 'OTHER') return -1;
-                  return this.props
-                    .t(`expense:${firstExpenseType.expense_translation_key}`)
-                    .localeCompare(
-                      this.props.t(`expense:${secondExpenseType.expense_translation_key}`),
-                    );
-                })
-                .map((type) => {
-                  return (
-                    <Grid
-                      item
-                      xs={4}
-                      md={3}
-                      lg={2}
-                      key={type.expense_type_id}
-                      style={{ marginBottom: '12px' }}
-                    >
-                      <div>
-                        <div
-                          style={
-                            selectedTypes.includes(type.expense_type_id)
-                              ? selectedStyle
-                              : unSelectedStyle
-                          }
-                          onClick={() => this.addRemoveType(type.expense_type_id)}
-                          className={styles.greenCircle}
-                        >
-                          <img
-                            src={iconMap[type.expense_translation_key]}
-                            alt=""
-                            className={styles.circleImg}
-                          />
-                        </div>
-                        <div className={styles.typeName}>
-                          {this.props.t(`expense:${type.expense_translation_key}`)}
-                        </div>
-                      </div>
-                    </Grid>
-                  );
-                })}
-            </Grid>
-          </div>
-          <div>
-            <div className={styles.manageCustomTypeLinkContainer}>
-              <IconLink
-                id="manageCustomExpenseType"
-                className={styles.manageCustomTypeLink}
-                icon={<BsGear className={styles.manageCustomTypeIcon} />}
-                isIconClickable
-                underlined={false}
-              >
-                {this.props.t('SALE.FINANCES.MANAGE_CUSTOM_EXPENSE_TYPE')}
-              </IconLink>
-            </div>
-            <Button
-              color={'primary'}
-              fullLength
-              onClick={() => this.nextPage()}
-              disabled={!selectedTypes.length}
-              className={styles.continueButton}
-            >
-              {this.props.t('common:CONTINUE')}
-            </Button>
-          </div>
-        </div>
+        <HookFormPersistProvider>
+          <PureFinanceTypeSelection
+            title={this.props.t('EXPENSE.ADD_EXPENSE.TITLE')}
```

Comment:
**question**: Why not doing the translation inside the component instead of passing title as a prop, since we're using `useTranslation` inside anyway ?

Do we plan to use different titles for this component depending of where it's called ?

Reply:
Yes, that's correct! We are going to use `PureFinanceTypeSelection` for revenue type as well.

Thank you so much for the translations Jalil!!!

Reply:
Of course, that makes sense … thank you for answering